### PR TITLE
Revert "Revert "Don't write to standard err""

### PR DESCRIPTION
**Pull request by bratseth**
Reverts vespa-engine/vespa#17078

Should be ok after https://github.com/vespa-engine/system-test/pull/1435

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/content/IndexedHierarchicDistributionValidator.java [hunk 1]
```diff
@@ -97,10 +97,10 @@ static public void validateThatReadyCopiesIsCompatibleWithRedundancy(String clus
         }
         if (totalReadyCopies % groupCount != 0) {
             throw new IllegalArgumentException(getErrorMsgPrefix(clusterName) + "Expected equal amount of ready copies per group, but " +
-                                totalReadyCopies + " ready copies is specified with " + groupCount + " groups");
+                                               totalReadyCopies + " ready copies is specified with " + groupCount + " groups");
         }
         if (totalReadyCopies == 0) {
-            System.err.println(getErrorMsgPrefix(clusterName) + "Warning. No ready copies configured. At least one is recommended.");
+            throw new IllegalArgumentException(getErrorMsgPrefix(clusterName) + "Warning. No ready copies configured. At least one is recommended.");
```
Comment (geirst) [inline config-model/src/main/java/com/yahoo/vespa/model/content/IndexedHierarchicDistributionValidator.java:103]: Perhaps we should re-phrase to "At least one is required"?

**Review by geirst — APPROVED** [review]
👍

File: config-model/src/main/java/com/yahoo/vespa/model/content/IndexedHierarchicDistributionValidator.java (hunk 1 above)
Comment (bratseth) [reply config-model/src/main/java/com/yahoo/vespa/model/content/IndexedHierarchicDistributionValidator.java]: done

**Comment by hmusum** [comment]
HierarchicDistributionTest::test_no_ready_copies started failing after this went in. Consider reverting or annotating/fixing test

**Pull request merged**